pkg/cableengine/healthchecker/statistics.go

```diff
@@ -64,7 +64,7 @@ func (s *statistics) update(rtt uint64) {
 		s.sum += rtt
 		oldMean := s.mean
 		s.mean = s.sum / (s.index + 1)
-		s.sqrDiff += uint64(((int64(rtt - oldMean)) * int64((rtt - s.mean))))
+		s.sqrDiff += uint64((int64(rtt - oldMean)) * int64(rtt - s.mean))
```

Comment:
To pass our linting, the spaces around the `-` operators need to be removed; in addition, there’s another set of parentheses which can be removed:

```suggestion
		s.sqrDiff += uint64(int64(rtt-oldMean) * int64(rtt-s.mean))
```

Reply:
Ok, thanks